docs/webhooks/webhooks.mdx

```diff
@@ -53,15 +57,16 @@ Lotus uses a service called Svix to handle the complexity behind webhooks, most
 ```
 
 <ParamField body="event_type" type="string (WebhookTriggerEventEnum)" required>
-  Enum: `"invoice.created"` `"invoice.past_due"` `"invoice.paid"`
-  `"usage_alert.triggered"` `"customer.created"`
+  Enum: `"customer.created"` `"invoice.created"` `"invoice.past_due"` `"invoice.paid"`
+  `"subscription.created"` `"usage_alert.triggered"`
 </ParamField>
 
 <ParamField body="payload" type="object" required>
   The payload of the webhook. The structure of the payload depends on the event
-  type. For `invoice.*`, the response will be an
-  [invoice](../api-reference/invoices/get-invoice). For `customer.created`, the
-  response will be a [customer](../api-reference/customers/retrieve-customer).
+  type.
+  For `customer.created`, the response will be a [customer](../api-reference/customers/retrieve-customer).
+  For `invoice.*`, the response will be an [invoice](../api-reference/invoices/get-invoice).
+  For `subscription.created`, the response will be a [subscription](../api-reference/subscriptions/list-subscriptions).
```

Comment:
When browsing the api reference page, I couldn't find a request for getting a single subscription so I used the `/api/subscriptions/` endpoint instead.

Reply:
Perfect @djanda97 . It can just be an object of a subscription. The api/subscriptions is just a list of these subscriptions.

Reply:
Looks good. Here is the payload I'm getting
`{
  "attempt": 5,
  "created_at": "2023-03-07 04:12:44.938810+00:00",
  "properties": {
    "event_type": "subscription.created",
    "payload": {
      "addons": [],
      "auto_renew": true,
      "billing_plan": {
        "plan_id": "plan_77af5f277e2347048d313ca1c4843251",
        "plan_name": "Events + Recordings - Pro",
        "version": 1,
        "version_id": "plan_version_399d3c5b9da9439a8d9412c70866d459"
      },
      "customer": {
        "customer_id": "cust_deef1bed569343d2bc3e4614dc0e49ff",
        "customer_name": "MediumCompany 9cb1fc",
        "email": "3cc9e9350ae7458c8dfdf92f01478629@aca92ffe6ae84ce2ba961ddeadae0d03.com"
      },
      "end_date": "2023-04-06T23:59:59.999999+00:00",
      "fully_billed": false,
      "is_new": true,
      "start_date": "2023-03-07T04:12:44.158000+00:00",
      "subscription_filters": []
    }
  }
}`